docs/en/api/POSIX-API-Distributed-Cache.md

```diff
@@ -1,9 +1,9 @@
 ---
 layout: global
-title: FUSE-based POSIX API
-nickname: POSIX API
+title: FUSE-based POSIX API with Alluxio Distributed Cache
+nickname: POSIX API with Alluxio Distributed Cache
 group: Client APIs
-priority: 3
+priority: 4
```

Comment:
nit: There are only 3 files in the api dir so the priority shouldn't be 4?

Reply:
Add a new POSIX API with local cache doc so becomes 4 files